lib/awful/widget/calendar_popup.lua

```diff
@@ -246,9 +246,12 @@ end
 --
 -- @param widget Widget to attach the calendar
 -- @tparam[opt="tr"] string position Two characters string defining the position on the screen
+-- @tparam[opt={}] table args Additional options
+-- @tparam[opt=false] bool args.on_hover Show popup while mouse hover
```

Comment:
i am not sure i getting the question so better if you try on your own:

```lua
  awful.widget.calendar_popup.month():attach(WIDGET, nil, {on_hover=true})
```


actually the part of scenario which seems a bit weird to me:

1) mouse hovered on widget - calendar appears
2) left mouse down (still hovered) -- calendar disappers